File: JavaScript_Courses/JavaScript_Basics/Preparation/01_First_Steps_In_Coding_Exercise/04_Vacation_Books_List.js
function vacationBooksList(input) {
    let totalPages = Number(input[0]);
    let pagesPerHour = Number(input[1]);
    let totalDays = Number(input[2]);

    let hoursNeeded = totalPages / pagesPerHour;
    let hoursPerDay = hoursNeeded / totalDays;

    console.log(hoursPerDay);
}

// vacationBooksList(["212", "20", "2"]);
// vacationBooksList(["432", "15", "4"]);

File: Python_Courses/Python_Basics/Python_PB_2025_01/01_First_Steps_in_Coding_Exercise/04_Vacation_books_list.py
# 1.	Брой страници в текущата книга – цяло число в интервала [1…1000]
total_pages = int(input())
# 2.	Страници, които прочита за 1 час – цяло число в интервала [1…1000]
pages_per_hour = int(input())
# 3.	Броят на дните, за които трябва да прочете книгата – цяло число в интервала [1…1000]
days = int(input())

total_hours_needed = total_pages // pages_per_hour
# print(total_hours_needed)
hours_per_day = total_hours_needed / days

print(hours_per_day)
